waiter/integration/waiter/token_request_test.clj

```diff
@@ -213,40 +252,92 @@
           (finally
             (delete-token-and-assert waiter-url token)))))))
 
+(deftest ^:parallel ^:integration-fast test-token-sync-unaffected-by-run-as-user-permissions
```

Comment:
But we're not calling it sync-ing elsewhere, we're calling it "update-mode admin"